Integrations/integration-Fortigate.yml

```diff
@@ -0,0 +1,1381 @@
+        res = session.get(
+            BASE_URL + target_url,
+            verify=USE_SSL
+        )
+        logout(session)
+        if res.status_code == 200:
+            return True
+        return False
+
+
+    def login():
+        """
+        Due to token not providing the right level of access, we are going to create a session and inject into its headers the csrf token provided with the service.
+        This won't work with usual requests as the session must be kept alive during this time.
+        """
+        # create session.
+        session = requests.session()
+        url_suffix = '/logincheck'
+        params = {
+            'username': USER_NAME,
+            'secretkey': PASSWORD,
+            'ajax': 1
+        }
+        session.post(SERVER+url_suffix, data=params, verify=USE_SSL)
+        # check for the csrf token in cookies we got, add it to headers of session or else we can't perform HTTP request that is not get.
+        for cookie in session.cookies:
+            if cookie.name == 'ccsrftoken':
+                csrftoken = cookie.value[1:-1] # strip quotes
+                session.headers.update({'X-CSRFTOKEN': csrftoken})
+        # Simply make sure the connection is valid and we can continue.
+        login_check = session.get(SERVER + "/api/v2/cmdb/system/vdom",verify=USE_SSL)
+        login_check.raise_for_status()
+        return session
+
+
+    def logout(session):
+        """
+        Due to limited amount of simultaneous connections we log out after each API request. Simple post request to /logout endpoint without params.
+        """
+        url_suffix = '/logout'
+        params = {}
+        session.post(SERVER+url_suffix, data=params, verify=USE_SSL)
+
+    ''' COMMANDS + REQUESTS FUNCTIONS '''
+
+
+    def test_module():
+        """
+        Perform basic login and logout operation, validate connection.
+        """
+        session = login()
+        logout(session)
+        return True
+
+    def get_addresses_command():
+        headers = []
+        contents = []
+        context = {}
+        addresses_context = []
+        address =  demisto.args().get('address')
+        name =  demisto.args().get('name', '')
+
+        addresses = get_addresses_request(address, name)
+        for address in addresses:
+            subnet = address.get('subnet')
+            if subnet:
+                subnet = subnet.replace(" ", "-")
+            contents.append({
+                'Name': address.get('name'),
+                'Subnet': subnet,
+                'StartIP': address.get('start-ip'),
+                'EndIP': address.get('end-ip')
+            })
+            addresses_context.append({
+                'Name': address.get('name'),
+                'Subnet': subnet,
+                'StartIP': address.get('start-ip'),
+                'EndIP': address.get('end-ip')
+            })
+
+        context['Fortigate.Address(val.Name && val.Name === obj.Name)'] = addresses_context
```

Comment:
Shouldn't we push also to `IP.` according to our content standard ?

Reply:
I can't actually think what would be the use of of pushing this data under IP.